src/compas_view2/shaders/120/arrow.vert

```diff
@@ -23,7 +24,7 @@ void main()
     vec4 start_position = projection * viewworld * transform * vec4(position, 1.0);
     vec4 end_position = projection * viewworld * transform * vec4(position + direction, 1.0);
     direction_2d = get_xy(end_position) - get_xy(start_position);
-    gl_PointSize = 100;
+    gl_PointSize = size;
```

Comment:
perhaps it would be a good idea to define meaningful presets for this. for example, `size = 1` set by the user becomes `100`, `size = 2` becomes `500` and `size = 3` becomes `2000`. like small, medium, large...